intro-Linux/ch_getting_started.tex

```diff
@@ -158,6 +175,14 @@ \subsection{Using non-defined variables}
 
 The moral here is: \strong{be very careful to not use empty variables unintentionally}.
 
+\strong{Tip for job scripts: use \texttt{set -e -u} to avoid using empty variables accidentally.}
+
+The \texttt{-e} option will immediately stop the script if any command fails and
```

Comment:
Using `-e` will not stop the script, it will result in the script stopping if ... ;)

Same below